shared/js/ui/models/site.es6.js

```diff
@@ -224,7 +224,7 @@ Site.prototype = window.$.extend({},
         getMajorTrackerNetworksCount: function () {
             // console.log('[model] getMajorTrackerNetworksCount()')
             // Show only blocked major trackers count, unless site is allowlisted
-            const trackers = this.protectionsEnabled ? this.tab.trackers : this.tab.trackersBlocked
+            const trackers = !this.protectionsEnabled ? this.tab.trackers : this.tab.trackersBlocked
```

Comment:
Happy to make this:
```suggestion
            const trackers = this.protectionsEnabled ? this.tab.trackersBlocked : this.tab.trackers
```

As imo this makes it easier to read.